benchmark-service/benchmark/benchmark.py

```diff
@@ -272,6 +247,93 @@ async def update_commits(app):
     }
 
 
+async def update_commit(app, sha):  # pylint: disable=unused-argument
+    github_client = app['github_client']
+    batch_client = app['batch_client']
+    gs_reader = app['gs_reader']
+
+    file_path = f'{BENCHMARK_RESULTS_PATH}/{sha}.json'
+
+    request_string = f'/repos/hail-is/hail/commits/{sha}'
+
+    gh_commit = await github_client.getitem(request_string)
```

Comment:
What happens if the commit doesn't exist? In a future PR, can we fix this? I have some ideas on how this should be done.